LICENSE

```diff
@@ -1,7 +1,7 @@
 -------------------------------------------------------------------- 
-                 The Xdebug License, version 1.02
-             (Based on "The PHP License", version 3.0)
-   Copyright (c) 2003-2018 Derick Rethans. All rights reserved.
+                 The Xdebug License, version 1.03
+             (Based on "The PHP License", version 3.01)
```

Comment:
IIRC, some distro (deb) have issue with 3.0

Reply:
Yeah, that's fine... pointless IMO, but fine :-)